src/pilot/config_dashboard.py

```diff
@@ -328,15 +267,15 @@ def prep_ceph_hosts(dashboard_node, ceph_nodes):
         with open(tmp_hosts, "r") as f:
```

Comment:
I dont see tmp_hosts defined there ?

Reply:
See line 263, tmp_hosts = os.path.join("/tmp", "hosts-{}".format(node.fqdn))

Reply:
thank you.. git hub collapsed/was hiding it